### DISCUSSION TITLE
Sprites from 8-Bit Sonic Platformers

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#124] [Q&A]
Can I use the sprites and backgrounds from the 8-bit Sonic Platforming games for the Master System/Game Gear in Sonic World Next?

**Comment by Techokami** [comment]
You can use whatever you want in your own fangame project... because it's your own fangame project. However you might encounter potential weirdness with physics if the scale is changed (since the Master System/Game Gear assets are *smaller* than the assets on the Genesis) so you will need to do some manual fine-tuning of the code. However that'd be for you to experiment and figure out.
